apps/metadata-editor-e2e/src/e2e/dashboard.cy.ts

```diff
@@ -1,38 +1,36 @@
 describe('dashboard', () => {
-  let originalList
-  let newList
+  let pageOne
   describe('pagination', () => {
-    it('should display different results on click on arrow', () => {
-      cy.visit('/')
+    it('should display different results on click on arrow and change url', () => {
+      cy.visit('/records/all')
+      cy.wait('@setPageSizeRequest')
```

Comment:
This appears to be undefined, did you forget to commit something?

Reply:
Uh, that was from an attempt to change the table size to 5 instead of 15, so that the test would not fail.

Reply:
Anyway I rebased and it looks like the pagination e2e test has some issues. Taking this opportunity to fix it.